unitex/src/fr/umlv/unitex/Unitex.java

```diff
@@ -115,8 +122,7 @@ public void actionPerformed(ActionEvent e1) {
 							@Override
 							public void run() {
 								ConfigManager.setManager(new ConfigManager());
-								Config.initConfig(args.length == 1 ? args[0]
-										: null);
+								Config.initConfig(args);
```

Comment:
Under Linux with Java 8 (1.8.0_121) the "UnitexToolLogger Not Found" dialog is now showing twice (after and before the splash screen). Steps to reproduce the problem:

```
mkdir /tmp/test
mv Unitex.jar Gramlab.jar /tmp/test
cd /tmp/test
java -jar Unitex.jar
```